util/ubuntu-setup

```diff
@@ -156,6 +160,11 @@ fi
 # install all of the packages listed in required_packages.txt
 apt-get install $ARGS $(sed 's/#.*//;/^$/d' $BASE/util/ubuntu-packages.txt)
 
+if [ "$SYSTEM" = "ubuntu-14.04" ]; then
+    sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.9 40
+    sudo update-alternatives --config g++
+fi
+
 if [ "$(clang --version | grep -Eo '[0-9]+\.[0-9]+' | head -n1)" \< "3.5" ]; then
     echo "-- Symlinking clang-3.5 to clang"
```

Comment:
Something like this is also a possibility, although this is probably overkill for this situation.